scripts/c4-component.puml

```diff
@@ -0,0 +1,42 @@
+@startuml Consensus
```

Comment:
The graph looks amazing. Can we add instructions somewhere that describe the tool we use for producing this diagram, and how to generate it? Also, we should add a checklist item to the PR template to make sure this diagram stays current.

Reply:
Good point! How about adding comments to the `.puml` file?

Reply:
Sounds good!